src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -384,11 +387,12 @@ TableHeroImage.propTypes = {
   party: node,
   confirmed: bool,
   heroName: string,
-  showPvgnaGuide: oneOfType([
+  showGuide: oneOfType([
     bool,
     PropTypes.number,
   ]),
-  pvgnaGuideInfo: shape({ url: string }),
+  guideUrl: string,
+  guideType: string,
```

Comment:
optional: you could probably infer the guideType from the Url if you want to save a prop